recipes/parsnp/meta.yaml

```diff
@@ -14,21 +14,18 @@ source:
     sha256: '{{sha256osx}}'                                           # [osx]
 
 build:
-  number: 4
-  skip: True  # [py2k]
+  number: 0
 
 requirements:
   build:
   host:
-    - zlib
-    - llvm-openmp  # [osx]
-    - openmp
```

Comment:
Since the build script only copies files, we don't have any `build`/`host` dependencies.
(Although it would be preferable if didn't just repackage binaries but build them here on Bioconda ;).)

Reply:
Ahh good call. Thanks. I had released a previous recipe that built the on the Bioconda container, however some of the binaries didn't work for some people on older systems. These lines are just leftover from when I tried that.

Reply:
> some of the binaries didn't work for some people on older systems.

Oh, interesting. Our build environment should be setup in a way to help with building compatible binaries (we currently target CentOS 6 as the minimum). This is actually one of the reasons we try to avoid to repackage binaries, because we usually encounter that prebuilt binaries are less compatible than the ones we build ;).

Reply:
Yea it could very well be an issue on our side. The build process for the package is fairly old and dealing with libMuscle is always a hassle. I'll try to go back to the built release and replicate some of the runtime errors and open them as issues.